src/sentry/api/endpoints/sentry_app_installation_details.py

```diff
@@ -19,3 +20,22 @@ def delete(self, request, installation):
             request=request,
         )
         return Response(status=204)
+
+    def put(self, request, installation):
+        serializer = SentryAppInstallationSerializer(
+            installation,
+            data=request.data,
+            partial=True,
```

Comment:
I don't think you need the `partial=True` here since we expect the status to be in the `request.data` and that's the only thing you are validating

Reply:
It might be good to keep it, though. Updating is almost always going to allow only a portion of the required data (opposed to requiring _all_ fields).

Reply:
I put it back